_commands/basics/echo.md

```diff
@@ -0,0 +1,68 @@
+---
+---
+
+echo
+--
+
+`echo` is used to output whatever it is given into stdio. Often useful see the value of enviromental variables.
+
+~~~ bash
+$ echo "Hello World!"
+Hello World!
+$ echo $USER
+<your username>
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `echo -e [string...]`
+
+<pre>
+$ echo "Hello \tWorld!\n"
+Hello \tWorld\n 
+$ echo -e "Hello \tWorld!\n"
+Hello   World 
+
+$ echo -e "\033[31mHello \033[36mWorld\033[m"  
+<span style="color:red;">Hello</span><span style="color:cyan;"> World</span>
+$ echo "\033[31mHello \033[36mWorld\033[m"
+\033[31mHello \033[36mWorld\033[m
```

Comment:
Can you add an extra example here that uses `tput` instead of manual escape characters? This is a good example, but it's generally a good idea to use `tput` instead of raw escape codes as it's more portable and way more readable.

Try something like:
`echo "$(tput setaf 1)Hello $(tput setaf 6)World$(tput sgr0)"`